yarn.lock

```diff
@@ -6858,6 +6816,15 @@ simple-get@^4.0.0:
     once "^1.3.1"
     simple-concat "^1.0.0"
 
+simple-git@^3.27.0:
+  version "3.27.0"
+  resolved "https://registry.yarnpkg.com/simple-git/-/simple-git-3.27.0.tgz#f4b09e807bda56a4a3968f635c0e4888d3decbd5"
+  integrity sha512-ivHoFS9Yi9GY49ogc6/YAi3Fl9ROnF4VyubNylgCkA+RVqLaKWnDSzXOVzya8csELIaWaYNutsEuAhZrtOjozA==
+  dependencies:
+    "@kwsites/file-exists" "^1.1.1"
```

Comment:
Concerning dependency: no-commits in 5 years https://github.com/kwsites/file-exists & similarly to the `promise-deferred` library (see next line).

`simple-git` as a whole hasn’t had any commits in 4 years https://github.com/simple-git-js/simple-git/

It’s one thing if tools are based in a mature stack that doesn’t change. But all these tools are based on node & npm. And as we all now, that stack has been constantly changing. I’m not sure going from a package that last saw maintenance 6 years ago to one that is 4 years ago is a big enough improvement? (Also the debug library is a little weird)

My 2cents: I’d expect new libraries we adopt have a story about ESM (& CJS still), otherwise we’re piling up more work in front of maintaining danger-js in the next year or so.

Reply:
That's not the dependency: https://www.npmjs.com/package/simple-git is https://github.com/steveukx/git-js

Reply:
Oh!

Reply:
I’m much less worried about simple-git now, thanks @orta! My questions about the transitive deps are still there, but active maintenance on simple-git is at least reassuring

Reply:
For me, the biggest flaw in this PR is that `parse-git-config` did one simple thing in 200 lines - this new dependency is massive in comparison because it implements every git command and everyone has to get all this stuff so we can just look at a list of remotes in `danger init`

Why did you choose it? Did you look for a smaller dependency @ryanb93 ?

Reply:
Honestly, it just came up as the first library that could implement this functionality - but you're correct, it's a heavy dependency to bring in for only a single task. I'm not sure how you feel about using `execSync` but we could remove these dependencies entirely and use something like:

```typescript
const remoteUrl = execSync('git config --get remote.origin.url').toString().trim();
```

Reply:
Yeah, I think that's pretty reasonable 👍🏻